bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Partners.scala

```diff
@@ -22,12 +24,30 @@ object Partners {
   // REL-2248 Contains a list of partners that are not allowed on joint proposals
   val jointProposalNotAllowed = List[NgoPartner](NgoPartner.KR, NgoPartner.AU)
 
-  val ftPartners:Seq[(Option[NgoPartner], String)] = {
-    (None -> "None") :: NgoPartner.values.toList.map(p => Some(p) -> Partners.name.getOrElse(p, ""))
+  // REL-2670 A partner affiliation for an FT proposal can be either Ngo or Exchange (Subaru)
+  type FtPartner = Option[NgoPartner \/ ExchangePartner]
```

Comment:
Keck and CFHT are also an exchange partners and yet not an FT partner.  Perhaps we should make a new `FtExchangePartner` type that includes only Subaru?